### PR TITLE
Feature/model conversion

### PR DESCRIPTION
**Pull request by MatPiech**
Add model conversion notebook (from PyTorch to TF Lite Micro) to data_processing directory.

### CODE REVIEW

File: data_processing/notebooks/model_conversion.ipynb [hunk 1]
```diff
@@ -0,0 +1,1022 @@
...
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "b844dd84",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "def representative_dataset():\n",
+    "    df = pd.read_csv('../data_processing/example_data/one_person_close.csv', delimiter=',', skiprows=[0], index_col=0)\n",
```
Comment (przemyslaw-aszkowski) [inline data_processing/notebooks/model_conversion.ipynb:819]: How about taking more representative data, e.g. one of training dataset files? And changing row temperatures to normalized values (because the model uses normalized values as input)

**Review by przemyslaw-aszkowski — APPROVED** [review]
